os/se/ameba/Kconfig

```diff
@@ -44,3 +44,20 @@ config HW_SE_STORAGE
 	---help---
 		SE supports Secure Stroage
 
+config SE_FACTORY_CERT_ADDRESS
+	hex "Factoty Cert Address"
+	default 1C0000
```

Comment:
Please add '0x' at the first of the address. If not, it cannot work properly.

Reply:
Thank for pointing out the mistake, modified.